CHANGELOG.md

```diff
@@ -4,8 +4,14 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+- Support for 1.15.0 ([#125])
+- Sensitive property key is setable via a secret ([#125])
+
 ### Changed
 
+- Removed support for 1.13.2 ([#125])
+- Added/removed some default config settings that changed from 1.13 to 1.15 ([#125])
```

Comment:
Missing the actual ref:
`[#125]: https://github.com/stackabletech/nifi-operator/pull/125`

Reply:
Oops